app/scripts/modules/netflix/tableau/application/appTableau.controller.ts

```diff
@@ -0,0 +1,30 @@
+import {IComponentController, ISCEService, module} from 'angular';
+
+import {AUTHENTICATION_SERVICE, AuthenticationService} from 'core/authentication/authentication.service';
+import {NetflixSettings} from '../../netflix.settings';
+import {Application} from 'core/application/application.model';
+
+require('../tableau.less');
```

Comment:
Didn't know we could do that w/ LESS files.  GTK and I like it way more betterer.